src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
+
+div {
+  background: #EEF0F1;
+}
+
+.App {
+  text-align: center;
+  font-family: 'Mulish', sans-serif;
+  min-height: 50rem;
+}
+
+.NavBar {
+  width: 100%;
+  height: 5rem;
+  text-align: center;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  flex-direction: row;
+  position: fixed;
+  top: 0;
+  background: #4267B2;
+  color: white;
+  min-width: 50%;
+  white-space: nowrap;
+  font-weight: 800;
+  z-index: 1;
+}
+
+#TeachLA {
+  width: 3.75rem;
+  height: 3.75rem;
+  left: 20px;
+  margin: 1.25rem;
+}
+
+#BiasByUs {
+  font-size: 3rem;
+  margin: 0.5625rem;
+}
+
+#currentCaseStudy {
+  font-size: 1.875rem;
+  margin: 1.25rem;
+  position: relative;
+  top: 0.3125rem;
+}
+
+button {
+  background-color: #EEF0F1;
+  width: 4.063rem;
+  min-width: 4.063rem;
+  height: 4.063rem;
+  border-radius: 50%;
+  border: none;
+  position: relative;
+  margin-left: auto;
+  margin-right: 4.375rem;
+  display: inline-block;
+}
+button:focus {
+  outline: none;
+}
```

Comment:
Ditto on styling `<button>`; instead, you should make a `.button` class.

Also, is there a reason that you're using the following styling? Seems slightly arbitrary to me

```
margin-left: auto;
margin-right: 4.375rem;
```

Reply:
Created .button class!

All of the items in the NavBar class are supposed to be aligned to the left side of the page because of `justify-content: left`, and I wanted the button alone to be on the right side of the page. I tried out `margin-left: auto` after a Google search on how to do this and it worked. If you have a suggestion for a better way to move the button to the right then please let me know!

Reply:
Oh I see, I see. In that case, I think this is good then (afaik this is how Bootstrap does it with `.ml-auto`). Thanks for clarifying!

(There are other ways to do this with Flexbox too, but this is solid!)